### PR TITLE
RK-11043 - adding github actions workflows + changing scripts to run on ubuntu

### PR DESCRIPTION
**Pull request by Goobensio**
-Removed .circleCI
-Added .github
-Changed scripts to run on ubuntu

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: validate-ci
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, reopened, edited, ready_for_review, synchronize]
+    branches-ignore:
+      - master
+      - '!*'
```
Comment (Gilwe) [inline .github/workflows/validate.yml:7]: Delete this part

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          sudo mkdir -p /tmp/helm/bin
+          sudo mkdir -p /tmp/helm/publish
+          sudo apt update -y
+          sudo apt install ca-certificates git openssh-server -y
+          cd /tmp/helm/bin
+
+      - name: Install helm 🚀
+        uses: azure/setup-helm@v1
+        with:
+          version: '2.8.1'
```
Comment (Gilwe) [inline .github/workflows/publish.yml:28]: 2.16.7

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -34,10 +49,25 @@ jobs:
...
+
+      - name: Publish & Deploy Charts 🚀
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.SONARIO_GH_TOKEN }}
+          publish_dir: /tmp/helm/publish/helm-charts
+          enable_jekyll: false
+          keep_files: true
+          user_name: "Github-Actions-CI"
+          user_email: ${{env.GITHUB_USERNAME}}@users.noreply.github.com
```
Comment (Gilwe) [inline .github/workflows/publish.yml:61]: why that email?

File: .github/workflows/validate.yml [hunk 4]
```diff
@@ -2,32 +2,42 @@ name: validate-ci
 on:
   pull_request:
     types: [labeled, unlabeled, opened, reopened, edited, ready_for_review, synchronize]
-    branches-ignore:
-      - master
-      - '!*'
   push:
     branches-ignore:
       - master
       - '*'
```
Comment (Gilwe) [inline .github/workflows/validate.yml]: dont push on master?

File: .github/workflows/publish.yml (hunk 3 above)
Comment (Goobensio) [reply .github/workflows/publish.yml:61]: This is the syntax that we used earlier, looks like people use this email type when they are using a bot to push / publish.

File: .github/workflows/validate.yml (hunk 4 above)
Comment (Goobensio) [reply .github/workflows/validate.yml]: On master push we only run publish.yml workflow. Here i am making sure validate.yml does not run when a push is being made on master.

File: .github/workflows/publish.yml (hunk 2 above)
Comment (Goobensio) [reply .github/workflows/publish.yml]: Updated to 2.16.7

**Review by Gilwe — APPROVED** [review]

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**